frontend/packages/dev-console/src/components/formik-fields/field-types.ts

```diff
@@ -83,5 +84,5 @@ export interface SecretKeyRef {
 }
 
 export interface RadioButtonProps extends FieldProps {
-  options: { value: string; label: string }[];
+  options: { value: string; label: string | React.ReactNode }[];
```

Comment:
FWIW, `React.ReactNode` covers `strings`, `numbers`, and `React Elements`. No need to do an or between a string and a node.

Also, based on my other feedback, we'll need helperText as well:
```
{
  value: string;
  label: React.ReactNode;
  helperText?: React.ReactNode;
}
```

In my other examples, I exported out Options so I could use it in the other page, you'd need to make that a new type here to do that. It's not strictly necessary.

Reply:
done